common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbUtils.java

```diff
@@ -210,10 +210,22 @@ public static List<Byte> stringToBytes(ExecutionContext ctx, Object str, String
         }
     }
 
-    private static byte[] bytesFromList(List<Byte> bytesList) {
+    private static byte[] bytesFromList(List<?> bytesList) {
         byte[] bytes = new byte[bytesList.size()];
         for (int i = 0; i < bytesList.size(); i++) {
-            bytes[i] = bytesList.get(i);
+            if (bytesList.get(i) instanceof Integer) {
```

Comment:
extract `bytesList.get(i)` to a variable (Object ? ). This will make code much more readable.